site/content/en/docs/getting_started/product_tour/cvat-intro.md

```diff
@@ -0,0 +1,10 @@
+---
+title: 'Cvat intro'
+linkTitle: 'Cvat intro'
+weight: 1
+---
+
+In this video, we show how to use CVAT - how to sign up, upload your data, annotate it, and download it.
+This video is the first one in a series that’ll guide you on our product and its many features.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/qyvMyDJUCmA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Need to move the video into another youtube channel: https://www.youtube.com/channel/UCV3qghS05GjAI714ID97IuA